source/settings/telemetry.txt

```diff
@@ -0,0 +1,134 @@
+.. _compass-telemetry:
+
+===========================
+Configure Telemetry Options
+===========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+|compass-short| collects anonymous aggregated usage data to improve
+MongoDB products. |compass-short| collects this information by default,
+but you can disable this data collection.
+
+About this Task
+---------------
+
+|compass-short| telemetry tracks the following data:
+
+- The type of MongoDB deployment that you connect |compass-short| to.
+  For example, Enterprise Edition, Community Edition, or Atlas Data
+  Lake.
+
+- Actions that you perform in |compass-short|, such as creating
+  collections and indexes. Telemetry only tracks the general action
+  performed, and does not track field values or namespaces.
+
+- Errors.
+
+|compass-short| *does not* track:
+
+- IP addresses, hostnames, usernames, or credentials
```

Comment:
We actually do track the hostname on connection in Compass. We use this information to identify an associated Atlas org when processing our telemetry data.

Reply:
That makes sense! Updating.